scripts/zkai_d128_gate_value_projection_proof_input.py

```
@@ -483,26 +540,33 @@ def validate_payload(payload: Any) -> None:
+            "public_instance_commitment": payload.get("source_bridge_public_instance_commitment"),
+            "projection_input_row_commitment": payload.get("source_projection_input_row_commitment"),
+        }
+    )
+    if payload.get("validation_commands") != validation_commands_for_bridge(
+        {
+            "statement_commitment": payload.get("source_bridge_statement_commitment"),
+            "public_instance_commitment": payload.get("source_bridge_public_instance_commitment"),
+            "projection_input_row_commitment": payload.get("source_projection_input_row_commitment"),
+        }
+    ):
+        raise GateValueProjectionInputError("validation commands drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)